conf/messages.en

```diff
@@ -544,8 +544,8 @@ propertyIncomeAllowance.error.maxCapped.individual = The amount of property inco
 propertyIncomeAllowance.error.maxCapped.agent = The amount of property income allowance your client claims must not be more than £1,000
 propertyIncomeAllowance.error.outOfRange.individual = Enter an amount between £{0} and £{1}
 propertyIncomeAllowance.error.outOfRange.agent = Enter an amount between £{0} and £{1}
-propertyIncomeAllowance.checkYourAnswersLabel = Your share of property income allowance
-propertyIncomeAllowance.change.hidden = Your share of property income allowance
+propertyIncomeAllowance.checkYourAnswersLabel = Your Property Income Allowance claim
```

Comment:
Could pls include these changes in messages.cy file as well?

Reply:
They are there @arunkumar461 - tests would as you know if they weren't there no?